### PR TITLE
[14.0][FIX] sale_line_refund_to_invoice_qty, fix default value on aml

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

### CODE REVIEW

File: sale_line_refund_to_invoice_qty/migrations/14.0.2.0.1/post-fix-old-field-value.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        env.cr.execute(query, (reference_date, reference_date))
+        lines_with_problems = [data[0] for data in env.cr.fetchall()]
+        if lines_with_problems:
+            _logger.debug("total lines to reprocess: %s", str(len(lines_with_problems)))
+            sol_model.search(
+                [
+                    ("id", "in", lines_with_problems),
+                    ("product_id.type", "!=", "service"),
+                ]
+            )._get_invoice_qty()
```
Comment (MiquelRForgeFlow) [inline sale_line_refund_to_invoice_qty/migrations/14.0.2.0.1/post-fix-old-field-value.py]:
```suggestion
            sol_lines = sol_model.search(
                [
                    ("id", "in", lines_with_problems),
                    ("product_id.type", "!=", "service"),
                ]
            )
            _logger.debug("total lines to reprocess: %s", str(len(sol_lines)))
            sol_lines._get_invoice_qty()
```
🤔

**Review by AaronHForgeFlow — APPROVED** [review]
code review :+1:

**Review by JuanyDForgeflow — APPROVED** [review]

**Pull request merged**